bcipy/signal/tests/evaluate/test_evaluator.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import numpy as np
+import math
+from bcipy.signal.evaluate.evaluator import Evaluator
+from bcipy.signal.evaluate.rules import HighVoltage, LowVoltage
+from mockito import any, unstub, when
+from bcipy.signal.generator.generator import gen_random_data
+
+
+class TestEvaluator(unittest.TestCase):
+    """Test Evaluator init and class methods """
+
+    def setUp(self):
+        self.expected_high_voltage = True
+        self.expected_low_voltage = True
+        self.parameters = {
+            'high_voltage_value': 1,
+            'low_voltage_value': -1
+        }
+
+        self.evaluator = Evaluator(self.parameters,
+                                   self.expected_high_voltage,
+                                   self.expected_low_voltage)
+
+        self.channels = 32
+
+    def tearDown(self):
+        unstub()
+
+    def test_init_rules(self):
+        """Test init of ruleset. We expect that rules enabled in the
+        params are initialized as part of the ruleset"""
+
+        if self.expected_high_voltage:
```

Comment:
remove if statements and make separate tests

Reply:
that makes sense, I've separated them into two different tests!